mp2-v1/tests/common/rowtree.rs

```diff
@@ -91,6 +91,9 @@ impl TestContext {
             let id = row.secondary_index_column;
             // Sec. index value
             let value = row.secondary_index_value();
+            let multiplier = table.columns.column_info(id).multiplier;
+            // NOTE remove that when playing more with sec. index
+            assert!(!multiplier, "secondary index should be individual type");
```

Comment:
Unsure. i didn't test secondary index coming from single table. I guess it should work but i didn't. I would leave it just in case we try that thing and we dont realize it is not tested.